pkg/capabilities/triggers/mercury_trigger.go

```diff
@@ -54,10 +49,22 @@ type subscriber struct {
 // Mercury Trigger will send events to each subscriber every MaxFrequencyMs (configurable per subscriber).
 // Event generation happens whenever local unix time is a multiple of tickerResolutionMs. Therefore,
 // all subscribers' MaxFrequencyMs values need to be a multiple of tickerResolutionMs.
-func NewMercuryTriggerService(tickerResolutionMs int64, lggr logger.Logger) *MercuryTriggerService {
+func NewMercuryTriggerService(tickerResolutionMs int64, capName string, capVersion string, lggr logger.Logger) *MercuryTriggerService {
 	if tickerResolutionMs == 0 {
 		tickerResolutionMs = defaultTickerResolutionMs
 	}
+	if capName == "" {
+		capName = defaultCapabilityName
+	}
+	if capVersion == "" {
+		capVersion = defaultCapabilityVersion
+	}
+
+	var capInfo = capabilities.MustNewCapabilityInfo(
```

Comment:
Using Must is fine as a global, but we should probably avoid it here, since this could panic.